scripts/ndk-make.sh

```diff
@@ -15,6 +15,8 @@
 #
 #     adb shell uname -m
 #
+# Use an app like: https://f-droid.org/packages/com.kgurgul.cpuinfo/
```

Comment:
indeed I also like that app and have used it and recommended it to know the architecture of the phone